nemo/collections/tts/torch/tts_tokenizers.py

```diff
@@ -231,6 +231,77 @@ def __init__(
         )
 
 
+class GermanPhonemesTokenizer(BaseCharsTokenizer):
+    # fmt: off
+    PUNCT_LIST = (  # Derived from LJSpeech and "/" additionally
+        ',', '.', '!', '?', '-',
+        ':', ';', '/', '"', '(',
+        ')', '[', ']', '{', '}',
+    )
+    # fmt: on
+
+    def __init__(
+        self,
+        punct=True,
+        apostrophe=True,
+        add_blank_at=None,
+        pad_with_space=False,
+        non_default_punct_list=None,
+        text_preprocessing_func=german_text_preprocessing,
+    ):
+        """Deutsch char-based tokenizer.
```

Comment:
Char-based or phoneme-based?

Reply:
Good catch! will fix it in next revision.